src/scripts/modules/media/body/body.coffee

```diff
@@ -439,21 +441,26 @@ define (require) ->
         @jaxing = false
         @processCoachMath?()
 
-      # Update the hash fragment after the content has loaded
-      # to force the browser window to find the intended content
+      # Clear and replace the hash fragment after the content has loaded
+      # to force the browser window to find the intended content (as a side effect)
       jumpToHash = () =>
         if currentPage.get('loaded') and not @fragmentReloaded and window.location.hash
           @fragmentReloaded = true
           hash = window.location.hash
           window.location.hash = ''
           window.location.hash = hash
 
+          # scroll up by width of nav bar
+          # FIXME - make this dynamic
+          window.scrollBy(0, -145)
```

Comment:
Thanks for trying to fix this problem but I think this might break responsiveness. 

There was some discussion on how to handle this in general and I think the decision was to have the top nav bar work the same was [news](https://www.washingtonpost.com/news/posteverything/wp/2017/06/28/the-supreme-court-is-in-no-hurry-to-protect-voters-from-gerrymandering/#article-body) websites and GitHub do it (by hiding the nav bar when you scroll down and re-show it when you scroll up)

Can you remove this from this Pull Request?